Automattic/wp-calypso
# Add steps for the migration instructions

Related to #91777

## Proposed Changes

* Prepare the components for the instruction steps.
  * The integration to make it dynamic and work properly will come in a following PR, so we reduce complexity for a single PR.

## Testing Instructions



* Make sure you have the feature flag `migration-flow/new-migration-instructions-step` activated.
* Open the storybook for the Launchpad (`cd packages/launchpad && yarn storybook`).
* Check the new stories for "Expanded" and "Expanded With Action" and make sure it works properly.
* Create a new site, navigate through the whole flow until the launchpad (screen with the title _"Let's get ready to launch!"_), and make sure the checklist continues working properly as previously.
* Now create a new site going through the migration flow, until you get into the new migration instructions screen.
* Check that you see the steps (the navigation doesn't work yet).
* Check it with and without the `from` (you can just update the URL adding/removing it), and make sure you see the workaround when it doesn't have the from URL.

## Screenshots

#### With "from"

<img width="434" alt="Screenshot 2024-07-12 at 11 18 01" src="https://github.com/user-attachments/assets/bf5890f2-db94-4421-9904-05ae92d4e97f">

#### Without "from"

<img width="447" alt="Screenshot 2024-07-12 at 11 18 11" src="https://github.com/user-attachments/assets/37cdf7e7-f184-4b24-83e0-7ccaac178407">

## Pre-merge Checklist
